src/workloads/execution/VariadicAggregateExpressions.yml

```diff
@@ -0,0 +1,311 @@
+      Pipeline: [{$project: {add: { $add: *IntFields10}}}]
+
+- ExpressionAdd20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionAdd20
+      nopInPhasesUpTo: *MaxPhases
+      active: [4]
+      operationMetricsName: VariadicAggExpressionAdd20
+      Pipeline: [{$project: {add: { $add: *IntFields20}}}]
+
+- ExpressionConcat10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionConcat10
+      nopInPhasesUpTo: *MaxPhases
+      active: [5]
+      operationMetricsName: VariadicAggExpressionConcat10
+      Pipeline: [{$project: {concat: { $concat: ["$str1", "$str2", "$str3", "$str4", "$str5", "$str6", "$str7", "$str8", "$str9", "$str10"]}}}]
+
+- ExpressionConcat20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionConcat20
+      nopInPhasesUpTo: *MaxPhases
+      active: [6]
+      operationMetricsName: VariadicAggExpressionConcat20
+      Pipeline: [{$project: {concat: { $concat: ["$str1", "$str2", "$str3", "$str4", "$str5", "$str6", "$str7", "$str8", "$str9", "$str10", "$str11", "$str12", "$str13", "$str14", "$str15", "$str16", "$str17", "$str18", "$str19", "$str20"]}}}]
+
+- ExpressionConcatArrays10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionConcat10
+      nopInPhasesUpTo: *MaxPhases
+      active: [7]
+      operationMetricsName: VariadicAggExpressionConcatArrays10
+      Pipeline: [{$project: {concatArrays: { $concatArrays: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10"]}}}]
+
+- ExpressionConcatArrays20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionConcatArrays20
+      nopInPhasesUpTo: *MaxPhases
+      active: [8]
+      operationMetricsName: VariadicAggExpressionConcatArrays20
+      Pipeline: [{$project: {concatArrays: { $concatArrays: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10", "$arr11", "$arr12", "$arr13", "$arr14", "$arr15", "$arr16", "$arr17", "$arr18", "$arr19", "$arr20"]}}}]
+
+- ExpressionIfNull10:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNull10
+      nopInPhasesUpTo: *MaxPhases
+      active: [9]
+      operationMetricsName: VariadicAggExpressionIfNull10
+      Pipeline: [{$project: {ifNull: { $ifNull: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10", 0]}}}]
+
+- ExpressionIfNull20:
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: VariadicExpressionIfNull20
+      nopInPhasesUpTo: *MaxPhases
+      active: [10]
+      operationMetricsName: VariadicAggExpressionIfNull20
+      Pipeline: [{$project: {ifNull: { $ifNull: ["$arr1", "$arr2", "$arr3", "$arr4", "$arr5", "$arr6", "$arr7", "$arr8", "$arr9", "$arr10", "$arr11", "$arr12", "$arr13", "$arr14", "$arr15", "$arr16", "$arr17", "$arr18", "$arr19", "$arr20", 0]}}}]
+
+- ExpressionLet10:
```

Comment:
I'm not sure that we want to test let? If we do though, we should make sure that the "in" expression is small and doesn't reference most of the bound fields. That way, we can measure the cost of binding variables.

Reply:
Done.